projects/fsastorefrontlib/src/cms-components/seller-dashboard/seller-dashboard.component.html

```diff
@@ -71,7 +71,7 @@ <h5 class="text-center py-3 mb-0">
                 <div class="number notice semi-bold">
                   {{
                     customers?.entries?.length > 0
-                      ? customers?.entries?.length
+                      ? customers?.pagination?.totalCount
```

Comment:
This is not related to this bug. I noticed something new while fixing this. Instead of showing only customers' entries length which corresponds to the number of customers per page count (in this case it is 10 per page), with this we display all existing customers in our database for a particular seller.

Reply:
I know you maybe didn't write this one (it was probably me), but it seems chaining operator after `customers` is also redundant. Please check.

Reply:
I'll try just with `{{ customers?.pagination?.totalCount }}`, it should work in both cases.

Reply:
Yes, but my point was that I left `?` after customers by mistake, since it's checked in the first if condition above.